src/__tests__/components/Navbar.test.tsx

```diff
@@ -142,4 +178,24 @@ describe('Navbar', () => {
       expect(children.props.children[1].props.children).toEqual('actions.search')
     })
   })
+
+  describe('add new', () => {
+    let addNew = hospitalRunNavbar.find('.add-new')
+    let children = addNew.first().props().children as any
```

Comment:
let's make these variables scoped to each `it` test.

Reply:
✅ in 860fb8c